package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0-semantically-released",
   "description": "Custom jest matchers to test the state of the DOM",
   "main": "dist/index.js",
+  "types": ".",
```

Comment:
I don't think this actually does anything... `import '@testing-library/jest-dom/extend-expect';` from a TS file has always worked.

Again, the point of #123 was to cheat (a bit) and make the `extend-expect` typings available without needing any imports (i.e., make them "global" typings). But the only way to make them global typings is by moving them to DefinitelyTyped and defining them as global typings, rather than module typings.

This is something that the DT maintainers disagreed with, so barring some kind of TS compiler support for automatic global type resolution from types provided by npm packages, we are stuck.

Reply:
For me, this did make a difference. Without this, it didn't work - I checked a few times.

For me, this only worked otherwise if I did:
```ts
import '@testing-library/jest-dom/extend-expect.d'
```

This may be environmental.